docs/EditInDialogButton.md

```diff
@@ -51,24 +61,297 @@ const CompanyShow = () => (
 
-Check out [the `ra-form-layout` documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createindialogbutton-editindialogbutton-and-showindialogbutton) for more details.
+This is useful to link to a related record. For instance, the following button lets you show the author of a book:
+
+```jsx
+const EditAuthorButton = () => {
+  const book = useRecordContext();
+  return (
+    <EditInDialogButton resource="authors" id={book.author_id}>
+        ...
+    </EditInDialogButton>
+  );
+};
+```
+
+## `sx`
+
+Customize the styles applied to the Material UI `<Dialog>` component:
+
+{% raw %}
+```jsx
+const EditButton = () => (
+  <EditInDialogButton sx={{ backgroundColor: 'paper' }}>
+      ...
+  </EditInDialogButton>
+);
+```
+{% endraw %}
+
+## Redirection After Deletion
+
+If you use `<SimpleForm>` as child of `<EditInDialogButton>`, the default form toolbar includes a `<DeleteButton>`. And upon deletion, this button redirects to the current resource list. This is probably not what you want, so it's common to customize the form toolbar to disable the redirection after deletion:
+
+{% raw %}
+```tsx
+// src/CustomToolbar.tsx
+import { Toolbar, SaveButton, DeleteButton } from 'react-admin';
+
+export const CustomToolbar = () => (
+    <Toolbar sx={{ justifyContent: 'space-between' }}>
+        <SaveButton />
+        <DeleteButton redirect={false} />
+    </Toolbar>
+);
+
+// src/EmployerEdit.tsx
+import { Edit, SimpleForm, TextInput, ReferenceManyField } from 'react-admin';
+import { EditInDialogButton } from '@react-admin/ra-form-layout';
+import { CustomToolbar } from './CustomToolbar';
+
+const EmployerEdit = () => (
+    <Edit>
+        <SimpleForm>
+            ...
+            <ReferenceManyField target="employer_id" reference="customers">
+                <Datagrid>
+                    ...
+                    <EditInDialogButton fullWidth maxWidth="sm">
+                        <SimpleForm toolbar={<CustomToolbar />}>
+                            <TextInput source="first_name" />
+                            <TextInput source="last_name" />
+                        </SimpleForm>
+                    </EditInDialogButton>
+                </Datagrid>
+            </ReferenceManyField>
+        </SimpleForm>
+    </Edit>
+);
+```
+{% endraw %}
 
 ## Combining With `<CreateInDialogButton>`
 
-Below is an example of an `<Edit>` view, inside which is a nested `<Datagrid>`, offering the ability to **create**, **edit** and **show** the rows thanks to [`<CreateInDialogButton>`](./CreateInDialogButton.md), `<EditInDialogButton>` and [`<ShowInDialogButton>`](./ShowInDialogButton.md):
+Below is an example of an `<Edit>` view, inside which is a nested `<Datagrid>`, offering the ability to **create**, **edit** and **show** the rows thanks to [`<CreateInDialogButton>`](./CreateInDialogButton.md), `<EditInDialogButton>` and [`<EditInDialogButton>`](./EditInDialogButton.md):
```

Comment:
Not sure this replacement is intentional. You should keep `ShowInDialogButton` here IMO.